apps/libexec/node.py

```diff
@@ -400,9 +400,8 @@ def cmd_ctrl(args):
 
 	global wanted_names
 
-	if len(args) == 0:
-		print("Control without commands seems quite pointless. I'm going home")
-		print("Try 'mon node help' for some assistance")
+	if not args:
+		print("Try 'mon node help' for available commands.")
```

Comment:
I think it's worth letting the user know there s/he need to specify some command to run on the remote node. Also, `mon node ctrl` is used to execute any arbitrary bash cmd on a node, not the ones listed in `mon node help`

Reply:
Isn't that what happens when the tools exits with the message above, that user was notified that something in the command is missing? Or do you have an alternative suggestion for the message?

The most detailed help there is on `mon node ctrl` is being shown by `mon node help`:
```
# mon node help

Available commands in category node:
 add <name> type=[peer|poller|master] [var1=value] [varN=value]
   Adds a node with the designated type and variables.

 ctrl <name1> <name2> [--self] [--all|--type=<peer|poller|master>] -- <command>
   Execute <command> on the remote node(s) named. --all means run it on
   all configured nodes, as does making the first argument '--'.
   --type=<types> means to run the command on all configured nodes of
   the given type(s).
   --self makes the command also run on the local host.
   The first not understood argument marks the start of the command,
   but always using double dashes is recommended. Use single-quotes
   to execute commands with shell variables, output redirection or
   scriptlets, like so:
     mon node ctrl -- '(for x in 1 2 3; do echo $x; done) > /tmp/foo'
     mon node ctrl -- cat /tmp/foo

```

Reply:
I think the reference to `mon node help` is correct. I also think it could be helpful if we can pinpoint why the command failed.

In this case we need both an actual command and some kind of arguments (node name(s)/node type). Can we tell them apart here?

Reply:
What I mean is that `mon node help` lists a lot of commands which can be run by `mon node ...` such as tree, status, info etc. If I got this message I would assume that those are the commands which are to be run by `mon node ctrl`. 

An alternative suggestion. "No command given to execute.\n See mon node help for info on how to use mon node ctrl". Or something like that.